install_deps.sh

```diff
@@ -43,21 +43,13 @@ apt-get update && apt-get ${APT_GET_FLAGS} install \
     iproute2 \
     wget
 
-update-alternatives --install /usr/bin/clang clang /usr/lib/llvm-9/bin/clang 100
```

Comment:
If we remove these alternatives it will broke down docker build functionality.
So these are important for docker build.

Reply:
if there are many changes needs to be done in exisiting install_deps file than I will recommend you to create a new install_deps file specific to native build.